e2e/migrations/migrations_test.go

```diff
@@ -0,0 +1,185 @@
+package e2e
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/nomad/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+const sleepJobOne = `job "sleep" {
+	type = "batch"
+	datacenters = ["dc1"]
+	constraint {
+		attribute = "${meta.secondary}"
+		value     = 1
+	}
+	group "group1" {
+		restart {
+			mode = "fail"
+		}
+		count = 1
+		ephemeral_disk {
+			migrate = true
+			sticky = true
+		}
+		task "sleep" {
+			template {
+				data = "hello world"
+				destination = "/local/hello-world"
+			}
+			driver = "exec"
+			config {
+				command = "/bin/sleep"
+				args = [ "infinity" ]
+			}
+		}
+	}
+}`
+
+const sleepJobTwo = `job "sleep" {
+	type = "batch"
+	datacenters = ["dc1"]
+	constraint {
+		attribute = "${meta.secondary}"
+		value     = 0
+	}
+	group "group1" {
+		restart {
+			mode     = "fail"
+		}
+		count = 1
+		ephemeral_disk {
+			migrate = true
+			sticky = true
+		}
+		task "sleep" {
+			driver = "exec"
+
+			config {
+				command = "test"
+				args = [ "-f", "/local/hello-world" ]
+			}
+		}
+	}
+}`
+
+func isSuccess(execCmd *exec.Cmd, retries int, keyword string) (string, error) {
```

Comment:
Can you add a comment that this looks for the keyword **not** to be in the string.